src/components/verification-grid/verification-grid.ts

```diff
@@ -343,6 +351,20 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
       return;
     }
 
+    // MacOS uses the command key instead of the ctrl key for keyboard shortcuts
+    // e.g. Command + A should select all items
+    // The command key is defined as the "meta" key in the KeyboardEvent object
+    // therefore, we conditionally check if the meta key is pressed instead of
+    // the ctrl key if the user is on a Mac
+    // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey
+    //
+    // TypeScript thinks that userAgentData can be undefined because it is not
+    // yet implemented on Firefox
+    // however, we have polyfilled the userAgentData object so we can be sure
+    // that userAgentData is implemented
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const hasKeyboardModifier = navigator.userAgentData!.platform.includes("Mac") ? event.metaKey : event.ctrlKey;
```

Comment:
Naming: shift and control are also modifiers so this name is ambiguous.

```suggestion
    const hasToggleModifier = navigator.userAgentData!.platform.includes("Mac") ? event.metaKey : event.ctrlKey;
```